data/human/human missions.txt

```diff
@@ -6849,3 +6849,320 @@ mission "FW Refugees to Humanika"
+			choice
+				`	"In."`
+				`	"Out."`
+					goto out
+
+			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet> where we are going to film the conditions in a secret Syndicate sweatshop. Thing is, these aren't just poorly paid factory workers, but slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking we got ourselves jobs on site as private security. We know that one of the Syndicate's 'trusted contractors' are getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories, you will be going with an associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
+			`	Ale senses your concern about spying on pirates and the Syndicate. "Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system, so you'll need to be quick."`
+			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
+			`	You go back to the <ship>'s landing pad, and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`
+			choice
+				`	"I am Captain <first> <last>. Well met."`
+				`	"Hi, I'm <first>."`
+					goto loadup
+			`	They all frown at your formality.`
+
+			label loadup
+			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
+				accept
+
+			label out
+			`	"I'm sorry to hear that," she says between gulps of her smoothie. "I suppose it just wasn't meant to be." She finishes her "drink" with lightning speed and leaves.`
+				decline
+
+	on visit
+		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`
+
+
+mission "Adelita 3"
+	landing
+	name "Film slave transaction"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction. (You will be paid <payment> after you complete the entire job.)"
+	source "Maker"
+	destination "Hippocrates"
+	"apparent payment" 200000
+	infiltrating
+	to offer
+		has "Adelita 2: done"
+	passengers 1
+	cargo "stealth drones" 1
+	on offer
+		conversation
+			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to take off for <planet>.`
```

Comment:
```suggestion
			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to take-off for <planet>.`
```

Reply:
I'm not sure about this one. Maybe the dash would be needed if it were `prepare for take-off to <planet>`?

Reply:
I don’t think that changes the meaning, does it? In either case it’s a synonym for ‘launch’, unless I’m reading it wrong.

Reply:
If you have issues with it, use "Depart."

Reply:
```suggestion
			`You drop off Ale and River at the spaceport. They are visibly embarrassed to be in security uniforms, or perhaps uncomfortable with their disguises which make them virtually unrecognizable; Bunker can't stop laughing at them. Everyone wishes each other good luck as the two walk off, and you and Bunker go back to the <ship> and prepare to depart for <planet>.`
```